pynsee/utils/_wait_api_query_limit.py

```diff
@@ -54,7 +58,7 @@ def _wait_api_query_limit(query):
         qCount = qCount.loc[qCount["oneMin"] == True]
         n_query = len(qCount.index)
 
-        # print("n query in 1 min : %s" % n_query)
+        # logger.info("n query in 1 min : %s" % n_query)

```

Comment:
delete or uncomment (moving it to debug)? cc @hadrilec

Reply:
it should be deleted